proto/src/ground/v1beta1/survey.proto

```diff
@@ -86,3 +89,18 @@ enum Role {
   // User may also manage the survey, update locations of interest, etc.
   SURVEY_ORGANIZER = 3;
 }
+
+// Defines the survey's completation status.
+enum SurveyStatus {
+  // Default value when status is missing.
+  STATUS_UNSPECIFIED = 0;
+
+  // Survey doesn't exists yet.
+  UNSAVED = 1;
+
+  // Survey exists but a job or some tasks are missing.
```

Comment:
`// Indicates the survey organizer has not yet completed the survey creation flow.`